tests/e2e/specs/IntermediateMessageThrowEvent.spec.js

```diff
@@ -115,4 +115,20 @@ describe('Intermediate Message Throw Event', () => {
       expect(numberOfMessages).to.equal(1, 'More than 1 message element was found');
     });
   });
+
+  it('should retain message name after loading XML', () => {
+    addNodeTypeToPaper(intermediateMessageThrowEventPosition, nodeTypes.intermediateCatchEvent, 'switch-to-intermediate-message-throw-event');
+
+    typeIntoTextInput('[name=messageName]', messageName);
+    cy.get('[aria-label="Toolbar"]').click();
```

Comment:
I had to select something outside of the inspector panel to trigger the `focusout` event; otherwise it created a race condition with the Undo button click event.

Reply:
I think this should be added as a comment to the code. The risk is that no-one will remember this the next time this test is edited and this will become a recurring problem. A comment seems appropriate here.

Reply:
Could this be re-useable in utils? This could be handy.

Reply:
I don't know if I want to make it a util since it's a hack. So far this is the only place I've needed this; it can be pulled out into a util when there's 1-2 other places we're having to use it, but hopefully by then we'll figure out a way to handle the weird async nature of these event handlers.